packages/services/src/__tests__/aggregationEvent.test.ts

```diff
@@ -0,0 +1,165 @@
+import * as vckitService from '../vckit.service';
+import { uploadJson } from '../storage.service';
+import * as linkResolverService from '../linkResolver.service';
+import { IContext } from '../epcisEvents/types';
+import { Result } from '../types/validateContext';
+import * as validateContext from '../epcisEvents/validateContext';
+import * as helpers from '../epcisEvents/helpers';
+import { processAggregationEvent } from '../epcisEvents';
+import { publicAPI } from '../utils/httpService';
+
+jest.mock('../vckit.service', () => ({
+  issueVC: jest.fn(),
+}));
+jest.mock('../storage.service', () => ({
+  uploadJson: jest.fn(),
+}));
+jest.mock('../linkResolver.service', () => ({
+  registerLinkResolver: jest.fn(),
+  createLinkResolver: jest.fn(),
+  IdentificationKeyType: jest.fn(),
+}));
+
+const nlisidMock = '9988776600000';
+const uploadedAggregationEventLinkMock = `https://s3.ap-southeast-2.amazonaws.com/${nlisidMock}`;
+const aggregationEventDLRMock = `https://example.com/nlisid/${nlisidMock}?linkType=all`;
+const aggregationVCMock = {
```

Comment:
Just a question: why don't we put the mock value in the mocks file?

Reply:
I want to keep all aggregated event mock data in one place for easy viewing and to avoid complexity.